apps/web-giddh/src/app/user-details/user-details.component.ts

```diff
@@ -319,7 +323,33 @@ export class UserDetailsComponent implements OnInit, OnDestroy, AfterViewInit {
     }
 
     public clearAllSession() {
```

Comment:
Add function return Type and also write documentation

Reply:
DONE